### ISSUE TITLE
11.1 Symbol Browser dead

### ISSUE THREAD

**Issue by cdcmicro** [Komodo/KomodoEdit#3564]
### Short Summary

In IDE 11.1 on Windows 10, the Symbol Browser (SB) fails to display any symbols.  Instead it shows the "wait" bars forever.

### Steps to Reproduce

1. Install IDE 11.1 in non-default directory (IDE 10 already installed, but no 11.x)
1. Restart Windows
1. Launch IDE 11.1
1. Open some files (js, perl)
1. Everything looks good.  Symbols visible.
1. Customize panes, save
1. Close IDE 11.1 (wait a bit)
1. Open IDE 11.1
1. Open an existing project from the commandline `ko -n some-project.komodoproject`

### Expected results

Symbols displayed in SB

### Actual results

SB sits and spins forever.  Closing/restarting has no effect.  SB dead forever.

### Platform Information

Komodo IDE, version 11.1.0, build 91033, platform win32-x86.
Built on Wed May 30 10:04:55 2018.

Windows 10 x64 1709

### Additional Information

![screenshot_062218_081959_am](https://user-images.githubusercontent.com/13684992/41784606-127291b2-75f5-11e8-9225-c5ab27ae55fc.jpg)

[pystderr.log](https://github.com/Komodo/KomodoEdit/files/2128170/pystderr.log)

**Comment by cdcmicro** [comment]
[pystderr.log (after closing 11.1)](https://github.com/Komodo/KomodoEdit/files/2128204/pystderr.log)

**Comment by Naatan** [comment]
Hmm there's an error in the log that didn't quite make it:

```
[2018-06-22 07:52:28,223] [DEBUG] console: error: {}
```

Could you check your Console pane to see what it says? I'm interested in the message, file and line number.

Also, can you reproduce this issue in safe mode? `Help > Troubleshooting > Restart in Safe-mode` ?

**Comment by cdcmicro** [comment]
Safe mode seems to work as expected.  And the same error is always displayed in the console, safe mode or not.

    fileName: "chrome://komodo/content/sdk/prefs.js"
    lineNumber: 264
    message: Component returned failure code: 0x8000ffff (NS_ERROR_UNEXPECTED) [koIPreferenceRoot.getPref]

**Comment by Naatan** [comment]
Could you try Help > Troubleshooting > Reset CodeIntel Database ?

**Comment by cdcmicro** [comment]
OK, reset/rebuilt CodeIntel DB.  No change.

Closed/opened IDE.  No change.

**Comment by Naatan** [comment]
Very strange. Did you try opening some different files (and languages) to see if that made a difference?

**Comment by cdcmicro** [comment]
Yes.  There's no change when a new file is opened.  Tried js, perl, sql, html

Each opened file appears to log:

    [2018-06-22 10:20:06,555] [WARNING] console-logger: Key event not available on GTK2: key="u" modifiers="control,shift" (1) in chrome://komodo/content/tail/tail.xul:0
    [2018-06-22 10:20:43,394] [ERROR] console-logger: TypeError: n is undefined (2) in chrome://komodo/content/contrib/commonjs/underscore.js:5
    Traceback (most recent call last):
      File "chrome://komodo/content/contrib/commonjs/underscore.js", line 5, in

**Comment by Naatan** [comment]
Could you see if blowing away your prefs.xml makes a difference? (Make a backup first, and be sure to shutdown Komodo while you do this).

The location of this file is explained here: http://docs.activestate.com/komodo/11/manual/trouble.html#appdata_dir

**Comment by cdcmicro** [comment]
Wow.  I had high hopes for that one, but no change.

Here's the log.  The console still shows the same error.

[pystderr.log](https://github.com/Komodo/KomodoEdit/files/2128952/pystderr.log)

**Comment by cdcmicro** [comment]
Update...

When SB the only widget in the pane, SB initializes dead as originally described.

If I move another widget (Toolbox) into the pane with SB and restart, SB shows symbols as expected - after I click to switch to it.

Interestingly, the Console displays the previously-mentioned error in all cases.  Not sure if it's related.

**Comment by Naatan** [comment]
Huh, interesting bug. I was able to reproduce your issue by making the symbol browser the only widget in my right pane and then restarting Komodo.

I don't have a workaround for you other than to ensure it isn't the only widget in a pane. Apologies for the inconvenience.

**Comment by cdcmicro** [comment]
Thanks for helping me get to the root condition.  I see it's marked as a bug now.

As long as it's short-lived, sharing the pane is a reasonable inconvenience.

**Comment by Naatan** [comment]
Note I think it only matters on startup, so you could technically still move it to a standalone pane after starting Komodo. Not ideal I know, but we'll fix this in 11.1.1 (that's a lot of 1's).

**Comment by niki-sp** [comment]
When Symbol Browser is single window clocking on tab "Symbol Browser" is required to display all symbols.
